core/controllers/dashboard.py

```diff
@@ -190,8 +195,33 @@ def _get_intro_card_color(category):
             key=lambda x: (x['num_open_threads'], x['last_updated']),
             reverse=True)
 
+        for collection_summary in subscribed_collection_summaries:
+            if collection_summary is None:
+                continue
+
+            collections_list.append({
+                'id': collection_summary.id,
+                'title': collection_summary.title,
+                'category': collection_summary.category,
+                'objective': collection_summary.objective,
+                'language_code': collection_summary.language_code,
+                'last_updated': utils.get_time_in_millisecs(
+                    collection_summary.collection_model_last_updated),
+                'created_on': utils.get_time_in_millisecs(
+                    collection_summary.collection_model_created_on),
+                'status': collection_summary.status,
+                'community_owned': collection_summary.community_owned,
+                'is_editable': True,
```

Comment:
Don't think it's needed actually. Will remove. Will file issue for collection editor needing a read-only mode.